packages/kbn-es-query/src/kuery/ast/ast.test.ts

```diff
@@ -16,8 +16,7 @@ import { nodeTypes } from '../node_types';
 import { DataViewBase } from '../../..';
 import { KueryNode } from '../types';
 import { fields } from '../../filters/stubs';
-
-jest.mock('../grammar');
```

Comment:
This grammar file was not being updated as part of the build, and it doesn't need to be mocked any longer anyway.